### ISSUE TITLE
Firefox (Mac OS X)

### ISSUE THREAD

**Issue by iliade81** [kolgepratik/js.tableutils#6]
(demo and local testing) tableutils not working with Firefox 22.0 (Mac)

**Comment by kolgepratik** [comment]
Is there any particular functionality that is not working as expected or nothing seems to work ? I hadn't got a chance to test it on Mac and so there might be bugs. If there are any more details you could provide, then it would be helpful. Meanwhile I will test it on Mac and fix the bugs that I encounter.

**Comment by iliade81** [comment]
you can check this video i've made ;) here : http://iliade.be/jquery.swf
Only the "head fix" works in firefox.
i tried in Firefox under Windows 7 (VMware), and it doesn't works :/

**Comment by kolgepratik** [comment]
Yes, I figured out there are some CSS Styles that are specific to IE that i have used in the plugin which causes Firefox to drop them, which in turn breaks the plugin. I am fixing this and will be releasing the fix in a few days. :)
